### PR TITLE
add 307,308 to redirect assertion chaijs/chai-http#147

### PR DESCRIPTION
**Pull request by tonymet**
- add 307, 308 codes to `redirect` assertion.  Also fixes `redirectTo`
- update docs and tests

### CODE REVIEW

File: lib/http.js [hunk 1]
```diff
@@ -244,13 +244,13 @@ module.exports = function (chai, _) {
...
   Assertion.addProperty('redirect', function() {
-    var redirectCodes = [301, 302, 303]
+    var redirectCodes = [301, 302, 303, 307, 308]
       , status = this._obj.status
       , redirects = this._obj.redirects;
 
     this.assert(
         redirectCodes.indexOf(status) >= 0 || redirects && redirects.length
-      , "expected redirect with 30{1-3} status code but got " + status
+      , "expected redirect status code but got " + status
```
Comment (vieiralucas) [inline lib/http.js]: I think this is fine, just want to share some ideas:

What about something like?
`"expected redirect with 30x status code but got " + status`

Comment (tonymet) [inline lib/http.js]: yeah that's clearer honestly couldn't think of a good error message so that's a good one

**Comment by tonymet** [comment]
not sure why nodev6 is not building in travis could you re-trigger the build -- i think it's an env issue

```
Bad response: 422 {"message":"Couldn't find a repository matching this job.","error":true}
npm ERR! Test failed.  See above for more details.
```

**Comment by tonymet** [comment]
great work thanks guys for accepting the PR and making the communication really clear.

**Pull request merged**